extensions/eclipse-che-theia-remote-impl-k8s/src/node/k8s-telemetry-service-impl.ts

```diff
@@ -0,0 +1,65 @@
+/**********************************************************************
+ * Copyright (c) 2021 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ ***********************************************************************/
+
+import { EventProperty, TelemetryClient } from '@eclipse-che/workspace-telemetry-client';
+
+import { TelemetryService } from '@eclipse-che/theia-remote-api/lib/common/telemetry-service';
+import { injectable } from 'inversify';
+
+@injectable()
+export class K8sTelemetryServiceImpl implements TelemetryService {
+  private readonly telemetryClient: TelemetryClient | undefined;
+
+  constructor() {
+    if (process.env.DEVWORKSPACE_TELEMETRY_BACKEND_PORT === undefined) {
+      console.error('Unable to create Che API REST Client: "CHE_WORKSPACE_TELEMETRY_BACKEND_PORT" is not set.');
```

Comment:
~`CHE_WORKSPACE_TELEMETRY_BACKEND_PORT`~ -> `DEVWORKSPACE_TELEMETRY_BACKEND_PORT`

Reply:
👍  will fix typo but it's not clear how to grab telemetry in devworkspace 2